pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -336,15 +336,15 @@ func getAppliedMcRemediations(r *ReconcileComplianceRemediation, rem *compv1alph
 //
 // taken from MachineConfigOperator
 func mergeMachineConfigs(configs []*mcfgv1.MachineConfig, name string, roleLabel string) *mcfgv1.MachineConfig {
-	mergedMc := mcfgv1.MergeMachineConfigs(configs, "")
+	mergedMc, err := mcfgcommon.MergeMachineConfigs(configs, "")
 
-	if mergedMc == nil {
+	// FIXME(jaosorior): Handle errors
+	if err != nil {
 		return nil
 	}
 
-	// NOTE(jaosorior): If no version was set (for some reason) lets just add a default
-	if mergedMc.Spec.Config.Ignition.Version == "" {
```

Comment:
Are we now relying on the MCs from the content to have the `Ignition.Version` set?

Reply:
yes... we probably need to write unit tests in CaC to ensure this.